playground/frontend/playground_components/lib/src/repositories/code_repository.dart

```diff
@@ -35,9 +35,11 @@ const kProcessingStartedText = 'The processing has started\n';
 
 // TODO(alexeyinkin): Rename. This is not a repository but a higher level client.
 class CodeRepository {
-  final CodeClient _client;
+  final CodeClient client;
```

Comment:
Client changed to public here just because of one method in `build_metadata.dart`. I suggest to keep this field incapsulated and turn back to private

Reply:
`+`